lib_standard_app/main.c

```diff
@@ -38,40 +38,34 @@ WEAK __attribute__((section(".boot"))) int main() {
 
     os_boot();
 
-    for (;;) {
-        // Initialize the UX system
-        UX_INIT();
+    BEGIN_TRY {
+        TRY {
+            UX_INIT();
```

Comment:
Do we remove this ?

Reply:
It's still used for NBGL, and can be a placeholder for futur changes?
So I'd say we should keep it.